sensu-plugins-strongswan.gemspec

```diff
@@ -39,6 +39,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake',                      '~> 10.0'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
   s.add_development_dependency 'rspec',                     '~> 3.1'
-  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
+  s.add_development_dependency 'rubocop',                   '~> 0.53'
```

Comment:
We are trying to update all the plugins to the same version:
```
 ~> 0.51.0
```

https://github.com/sensu-plugins/community/issues/77